tornjak-frontend/src/components/tornjak-helper.js

```diff
@@ -0,0 +1,53 @@
+import { Component } from 'react';
+import SpiffeHelper from './spiffe-helper';
+
+class TornjakHelper extends Component {
+  constructor(props) {
+    super(props);
+    this.SpiffeHelper = new SpiffeHelper();
+    this.state = {
+    };
+  }
+  
+  // detailsLink takes in selectedRows/ specified data and entity
+  // returns a specfic routing link/ URL for an entity
+  detailsLink(selectedRows, entity,) {
+    const dashboardDetailsLink = "/tornjak/dashboard/details/";
+    if (selectedRows.length !== 0) {
+      var detailsLink = dashboardDetailsLink + entity.toLowerCase() + "/" + encodeURIComponent(selectedRows.id); //encode URL since spiffeid contains special characters
+    }
+    return detailsLink;
```

Comment:
if the `if` statement here is not triggered, `detailsLink` would not be instantiated. Doesn't seem well formed